javascript/exercise1/ex3/booklist.js

```diff
@@ -0,0 +1,53 @@
+// The Recipe Card
+const obj = {
```

Comment:
apply naming convention pls. Naming with `obj`, `arr` are too general and confuse.